extensions/src/AWSSDK.Extensions.NETCore.Setup/Schema/AWSSDK.Extensions.NETCore.Setup.targets

```diff
@@ -0,0 +1,6 @@
+<Project>
```

Comment:
Used in the NuGet package for installing the JSON schema. I learned about this from the Aspire repository https://github.com/dotnet/aspire/blob/78609847e2759fd43bf13f2760ae30f39a08c932/src/Components/Common/Package.targets